tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -64,6 +64,15 @@ function install_artifacts() {
 	chmod +x /opt/kata/bin/*
 	[ -d /opt/kata/runtime-rs/bin ] && \
 		chmod +x /opt/kata/runtime-rs/bin/*
+
+	# Allow Mariner to use custom configuration.
+	if [ "${HOST_OS}" == "cbl-mariner" ]; then
+		config_path="/opt/kata/share/defaults/kata-containers/configuration-clh.toml"
+		clh_path="/opt/kata/bin/cloud-hypervisor-glibc"
+		sed -i -E 's|(enable_annotations) = .+|\1 = ["initrd", "kernel"]|' "${config_path}"
```

Comment:
this removes the default `enable_iommu` annotation from the list

Reply:
Fixed